contract init broken

<!-- Please answer these questions before submitting your issue. Thanks! -->

### What version are you using?

22.0.1

### What did you do?

```
stellar contract init .
cargo test
```

### What did you expect to see?

✅ Build success.

### What did you see instead?

❌ Build failure.

```
   Compiling hello-world v0.0.0 (contracts/hello-world)
error[E0599]: no method named `register` found for struct `Env` in the current scope
 --> contracts/hello-world/src/test.rs:9:27
  |
9 |     let contract_id = env.register(Contract, ());
  |                           ^^^^^^^^ method not found in `Env`
```

--- a/cmd/soroban-cli/src/utils/contract-workspace-template/Cargo.toml.removeextension
+++ b/cmd/soroban-cli/src/utils/contract-workspace-template/Cargo.toml.removeextension
@@ -5,7 +5,7 @@ members = [
 ]
 
 [workspace.dependencies]
-soroban-sdk = "21.0.0"
+soroban-sdk = "22"
 
 [profile.release]
 opt-level = "z"

